tests/app/notifications/test_validators.py

```diff
@@ -55,135 +55,64 @@ def enable_redis(notify_api):
     serialised_service = SerialisedService.from_id(sample_service.id)
     with set_config(notify_api, 'REDIS_ENABLED', False):
-        db_mock = mocker.patch('app.notifications.validators.services_dao')
-
+        mock_cache_key = mocker.patch('notifications_utils.clients.redis.daily_limit_cache_key')
         check_service_over_daily_message_limit('normal', serialised_service)
 
-        assert db_mock.method_calls == []
-
-
-@pytest.mark.parametrize('key_type', ['team', 'normal'])
-def test_check_service_message_limit_over_message_limit_fails(key_type, sample_service, mocker):
-    with freeze_time("2016-01-01 12:00:00.000000"):
-        mocker.patch('app.redis_store.get', return_value=None)
-        mocker.patch('app.notifications.validators.redis_store.set')
-
-        sample_service.restricted = True
-        sample_service.message_limit = 4
-        template = create_template(sample_service)
-        serialised_service = SerialisedService.from_id(sample_service.id)
-
-        for _ in range(5):
-            create_notification(template)
-        with pytest.raises(TooManyRequestsError) as e:
-            check_service_over_daily_message_limit(key_type, serialised_service)
-        assert e.value.status_code == 429
-        assert e.value.message == 'Exceeded send limits (4) for today'
-        assert e.value.fields == []
-        app.notifications.validators.redis_store.set.assert_called_with(
-            str(sample_service.id) + "-2016-01-01-count", 5, ex=3600
-        )
+        assert mock_cache_key.method_calls == []
 
 
 @pytest.mark.parametrize('key_type', ['team', 'normal'])
-def test_check_service_message_limit_in_cache_over_message_limit_fails(
-        notify_db_session,
-        key_type,
-        mocker):
-    with freeze_time("2016-01-01 12:00:00.000000"):
-        mocker.patch('app.redis_store.get', side_effect=[
-            None,  # The serialised service
-            5,  # The rolling count
-        ])
-        mocker.patch('app.notifications.validators.redis_store.set')
-        mocker.patch('app.notifications.validators.services_dao')
-
-        service = create_service(restricted=True, message_limit=4)
-        serialised_service = SerialisedService.from_id(service.id)
-        with pytest.raises(TooManyRequestsError) as e:
-            check_service_over_daily_message_limit(key_type, serialised_service)
-        assert e.value.status_code == 429
-        assert e.value.message == 'Exceeded send limits (4) for today'
-        assert e.value.fields == []
-        app.notifications.validators.redis_store.set.assert_called_once_with(
-            f'service-{serialised_service.id}',
-            ANY,
-            ex=ANY,
-        )
-        assert not app.notifications.validators.services_dao.mock_calls
+def test_check_service_message_limit_over_message_limit_fails(key_type, mocker, notify_db_session):
+    service = create_service(restricted=True, message_limit=4)
```

Comment:
Minor: the `restricted` setting is irrelevant for this test, so we don't need it (or we could parametrize on it if we're worried about it having an effect).